package.json

```diff
@@ -34,11 +39,15 @@
     "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
     "serve": "^11.3.2",
-    "workbox-cacheable-response": "^5.1.3",
-    "workbox-routing": "^5.1.3",
-    "workbox-strategies": "^5.1.3"
+    "typescript": "^4.3.5",
+    "workbox-cacheable-response": "^6.2.4",
+    "workbox-core": "^6.2.4",
+    "workbox-routing": "^6.2.4",
+    "workbox-strategies": "^6.2.4"
   },
   "devDependencies": {
+    "@types/history": "^4.7.9",
+    "@types/node": "^16.6.1",
```

Comment:
Do we need node types?

Reply:
Okay I see, we need it when accessing `process.env`!

Reply:
Looks like we don't! Done.